src/sentry/static/sentry/app/components/helpSearch.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t, tn} from 'app/locale';
+import Search from 'app/components/search';
+import HelpSource from 'app/components/search/sources/helpSource';
+import SearchResult from 'app/components/search/searchResult';
+import SearchResultWrapper from 'app/components/search/searchResultWrapper';
+import space from 'app/styles/space';
+import {IconWindow} from 'app/icons';
+
+type HelpResult = Parameters<
+  React.ComponentProps<typeof HelpSource>['children']
+>[0]['results'][0];
+
+type ResultItemProps = HelpResult & {
+  highlighted: boolean;
+  itemProps: any;
```

Comment:
isn't this itemProps an object?

Reply:
Oops, I'll leave a comment here. I didn't want to type this until we have more of the search components typed.